### ISSUE TITLE
pip install version is out of date, seller_query interface broken

### ISSUE THREAD

**Issue by PaulLuoLi** [akaszynski/keepa#90]
Hey folks; great package, but you need to update your pip package.

It's out of date and some of the functions are incorrect. 

Specifically, I was trying to call `seller_query(asin, storefront=True)`, and it returned an error

Upon digging, the version installed on my disk has this: 
```py
async def seller_query(self, seller_id, domain='US'):
```

While on GIT, the file is: 
```py
 async def seller_query(self, seller_id, domain='US', storefront=False, update=None):
```

**Comment by akaszynski** [comment]
I do need to send out a release.  I'll keep issue up until it's out.

**Comment by akaszynski** [comment]
Just released `keepa==1.2.0` on PyPI:
https://pypi.org/project/keepa/
